src/torch_ucc_sendrecv.cpp

```diff
@@ -63,19 +69,28 @@ torch_ucx_status_t torch_ucx_comm_init(
   worker_params.thread_mode = UCS_THREAD_MODE_MULTI;
   st = ucp_worker_create(comm->ctx, &worker_params, &comm->worker);
   if (st != UCS_OK) {
-    fprintf(stderr, "TorchUCC: failed to init ucp worker\n");
+    fprintf(
+        stderr,
+        "TorchUCC: failed to init ucp worker %s\n",
+        ucs_status_string(st));
     goto close_ctx;
   }
 
   worker_attr.field_mask = UCP_WORKER_ATTR_FIELD_THREAD_MODE;
   ucp_worker_query(comm->worker, &worker_attr);
   if (worker_attr.thread_mode != UCS_THREAD_MODE_MULTI) {
-    fprintf(stderr, "TorchUCC: Thread mode multi is not supported\n");
+    fprintf(
```

Comment:
here it's not an error and so there is no corresponding status string to print. I think ucs_status_string(st) here is not needed

Reply:
@Sergei-Lebedev fixed